client/src/components/form/form.js

```diff
@@ -41,12 +41,13 @@ const Label = styled.label`
 
 const Input = styled.input`
 outline: none;
-padding: 2.5%;
+padding: %;
```

Comment:
I noticed this doesn't have any units, did you want to remove this line completely?